app/scripts/views/mixins/resend-mixin.js

```diff
@@ -43,8 +43,8 @@ define([
 
     _updateResendButton: function () {
       var self = this;
-      // Hide the button after 4 attempts. Redisplay button after a delay.
-      if (self._attemptedSubmits === TOO_MANY_ATTEMPTS) {
+      // Hide the button after too many attempts. Redisplay button after a delay.
+      if (self._attemptedSubmits >= Constants.EMAIL_RESEND_MAX_TRIES) {
         self.logScreenEvent('too_many_attempts');
```

Comment:
> I'm leaning toward a stateless fxa-client (seems like it should only be concerned with throttling responses that come from the auth-server anyway) and probably moving it to some third place, i.e. a resend model that could be instantiated for each action that should be limited.

YAAAAS to all of the above!

This could be a two phase update:

1. Remove all of the state from fxa-client.js and into resend-mixin.js. 
1. Move the state from resend-mixin.js into a model.

Maybe the first is sufficient and we are happy at that point.

@eoger - what do you think?